src/components/Search.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
```

Comment:
Using 4 spaces of indentation here, this should be made more uniform with the rest of the project